### DISCUSSION TITLE
Can tflite extract text from an image?

### DISCUSSION THREAD

**Discussion by azkadev** [tensorflow/flutter-tflite#256] [Q&A]
can you make an example in your repo?

**Comment by jasperjonkhans** [comment]
TensorFlow Lite cannot extract text by itself; it is an inference runtime
 that executes a model. To perform OCR, you need a text-detection and
 text-recognition model that has been converted to TFLite, together with
 the model-specific preprocessing and postprocessing code.

 For a Flutter application, the simplest option is usually Google ML Kit
 Text Recognition. If you need a fully custom or offline pipeline, you can
 bundle compatible TFLite detection and recognition models and run them
 through this plugin. An example in this repository would therefore need to
 choose a specific OCR model because the interpreter alone does not provide
 OCR functionality.
